src/reducer/admin/index.js

```diff
@@ -1,12 +1,13 @@
 import { combineReducers } from 'redux';
-import resourceReducer from './resource';
+import { createSelector } from 'reselect';
+import resourceReducer, * as resourceSelectors from './resource';
```

Comment:
There was. I changed the naming of the selectors themselves where required and to avoid aliases that wouldn't mean anything.